tests/models/test_image.py

```diff
@@ -613,8 +717,8 @@ def test_image_from_decam_exposure(decam_example_file, provenance_base):
     # assert im.dec == -26.25
     assert im.ra != e.ra
     assert im.dec != e.dec
-    assert im.ra == 116.23984530727733
-    assert im.dec == -26.410038282561345
+    assert im.ra == 116.32126671843677
+    assert im.dec == -26.337508447652503
```

Comment:
I'm curious: what made the coordinates change?

Reply:
The code had NAXIS1 and NAXIS2 backwards in a wcs.pixel_to_world call in image.from_exposure.  NAXIS1 is width and NAXIS2 is height.  If you want to find the coordinates of x along the horizontal axis and y along the vertical axis, you need to do wcs.pixel_to_world(x, y).  But, if you want to find that pixel in the numpy data array, it's data[y, x].

Reply:
great!